src/styles/itemAttributes.ts

```diff
@@ -141,4 +141,35 @@ export class ItemAttributes {
 
     return `${size}%`;
   }
+
+  /**
+   * return the corresponding size to the value with a logarithmic function
+   * @param value
+   * @param lower
+   * @param higher
+   * @param extrema
+   */
+  public static getAutomaticRangeStyleLog(
+    value: number,
+    {max, min}: {max: number; min: number},
+    lower: EdgeWidthExtrema | NodeSizeExtrema,
+    higher: EdgeWidthExtrema | NodeSizeExtrema
+  ): string {
+    // apply an offset for all the values (including max) if min is smaller than one and set min to 1
+    if (min < 1) {
+      value += -min + 1;
```

Comment:
it is better to use absolute value than -min in case min is between 0 and 1.
```suggestion
      value += Math.abs(min)+ 1;
```